plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/VidarrProvenanceDto.java

```diff
@@ -0,0 +1,108 @@
+package ca.on.oicr.gsi.shesmu.vidarr;
+
+import ca.on.oicr.gsi.shesmu.plugin.Tuple;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/** Bean of provenance response 'results' from Vidarr */
```

Comment:
This seems like a duplicate of https://github.com/oicr-gsi/vidarr/blob/master/vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/api/ProvenanceWorkflowRun.java ?